server/install.sh

```diff
@@ -12,7 +12,7 @@
 apt-get update && apt-get install -yq yasm
 mkdir -p $GOPATH/src/github.com/minio
 cd $GOPATH/src/github.com/minio
-git clone -b deis --single-branch https://github.com/deis/minio-src.git minio
+git clone -b master --single-branch https://github.com/minio/minio.git minio
```

Comment:
+1 here too. we built off master for alpha since our release was already a "moving target". now that we're solidified a bit, we should freeze the minio code that we release.

also, if/when Minio makes a release, it also would come in prebuilt binary form, so we could simply download and use it.